fastlane_core/lib/fastlane_core/ipa_file_analyser.rb

```diff
@@ -17,6 +17,16 @@ def self.fetch_app_version(path)
       return nil
     end
 
+    # Fetches the app platform from the given ipa file.
+    def self.fetch_app_platform(path)
+      plist = self.fetch_info_plist_file(path)
+      platform = nil
+      platform = plist['DTPlatformName'] if plist
+      platform = "ios" if platform.nil?
+      platform = "ios" if platform == "iphoneos" # via https://github.com/fastlane/spaceship/issues/247
```

Comment:
@KrauseFx Line 25 yea, but I think we need to keep line 26 to set it back to ios if the platform value in the plist is iphoneos